### PR TITLE
whiteboard only, tests not passing

### PR DESCRIPTION
**Pull request by EllaSvete**
- [ x] Top-level README “Table of Contents” is updated
 - [ x] README for this challenge is complete
       - [ x] Summary, Description, Approach & Efficiency, Solution
       - [ x] Picture of whiteboard
       - [ x] Link to code
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

### CODE REVIEW

File: python/code_challenges/linked_list_zip.py [hunk 1]
```diff
@@ -1,2 +1,20 @@
...
+    current_b = b.head
+
+    while current_b and current_a:
+            a.insert_after(current_a.value, current_b.value)
+            current_a = current_a.next
+            if current_a.next:
+                current_a = current_a.next
+            current_b = current_b.next
+
+    if current_a is None or current_b is None:
```
Comment (aaron-imbrock) [inline python/code_challenges/linked_list_zip.py:14]: I'd use '==' in place of 'is' here. I don't believe it will cause an issue in this code block but they're not equivalent and may confuse you later.
```
if current_a == None or current_b == None:
```
The == operator compares the value or equality of two objects, whereas the Python is operator checks whether two variables point to the same object in memory.

File: python/code_challenges/linked_list_zip.py [hunk 2]
```diff
@@ -1,2 +1,20 @@
+from data_structures.linked_list import LinkedList
+
 def zip_lists(a, b):
-    pass
+    current_a = a.head
+    current_b = b.head
+
+    while current_b and current_a:
+            a.insert_after(current_a.value, current_b.value)
```
Comment (aaron-imbrock) [inline python/code_challenges/linked_list_zip.py:8]: Starting at line 95 in linked_list.py 

```
        current = self.head
        while current:
            if current.value is value:
                new_node.next = current.next
                current.next = new_node
                break
```
It looks like the value of current.value does not equal value causing the if block to fail. Since the if block exists inside of a while loop that's the cause of your infinite loop.

Comment (aaron-imbrock) [reply python/code_challenges/linked_list_zip.py:8]: Check that you're checking `current` and `current.value` at the correct places.
